papers/srinath/article.md

```diff
@@ -1,15 +1,15 @@
+  Gallstones, typically benign and harmless hardened deposits of digestive fluids, present in the gallbladder can cause severe painful complications if left untreated and can lead to removal surgery. [@noauthor_gallstones_2021] Quantitative Trait Loci (QTL) analysis can be used to find potential genetic links through the analysis of logarithm of the odds (LOD) scores which can indicate a possible connection between those loci on the mouse chromosome and phenotypic presentation of a trait [@myles2008quantitative] linked to gallstone susceptibility such as weight, presence, severity and liver weight. Analyses were performed using the R/QTL package on a cohort of mice, in an intercross breed and fed a high-fat diet. [@lyons1] Analyses were compared between augmented data sets (to possibly prevent overfitting) and an analysis run on a non-augmented data set.
 ---
 
 ## Introduction
-The gallbladder is an organ in the upper right portion of the abdomen, directly below the liver, that releases bile, a fluid that the liver produces, that digests fats. Bile is a solution of cholesterol, bilirubin, and bile salts. (@clevelandclinic_gallbladder:_nodate) There are two primary categories of gallstones, cholesterol and pigment stones. Cholesterol stones form due to a lack of balance of cholesterol, bilirubin, and bile salts in the bile. It can form due to excess bilirubin or cholesterol or a lack of bile salts. The cause of pigment stones is currently unknown, but they tend to develop in patients already suffering from cirrhosis, biliary tract infections, and hereditary blood disorders such as sickle cell anemia. (@noauthor_gallstones_2021)
+The gallbladder is an organ in the upper right portion of the abdomen, directly below the liver, that releases bile, a fluid that the liver produces, that digests fats. Bile is a solution of cholesterol, bilirubin, and bile salts. [@clevelandclinic_gallbladder:_nodate] There are two primary categories of gallstones, cholesterol and pigment stones. Cholesterol stones form due to a lack of balance of cholesterol, bilirubin, and bile salts in the bile. It can form due to excess bilirubin or cholesterol or a lack of bile salts. The cause of pigment stones is currently unknown, but they tend to develop in patients already suffering from cirrhosis, biliary tract infections, and hereditary blood disorders such as sickle cell anemia. [@noauthor_gallstones_2021]
```

Comment:
```suggestion
The gallbladder is an organ in the upper right portion of the abdomen, directly below the liver, that releases bile, a fluid that the liver produces, that digests fats. Bile is a solution of cholesterol, bilirubin, and bile salts [@clevelandclinic_gallbladder:_nodate]. There are two primary categories of gallstones, cholesterol and pigment stones. Cholesterol stones form due to a lack of balance of cholesterol, bilirubin, and bile salts in the bile. It can form due to excess bilirubin or cholesterol or a lack of bile salts. The cause of pigment stones is currently unknown, but they tend to develop in patients already suffering from cirrhosis, biliary tract infections, and hereditary blood disorders such as sickle cell anemia [@noauthor_gallstones_2021].
```